docs/guide/extending/extending_multifile_upload.md

```diff
@@ -112,7 +112,7 @@ Without all the following properties the Multi-file Upload will not work.
 
 ### Optional properties
 
-Optionally, the Multi-file Upload module can take a following list of props:
+Optionally, the Multi-file Upload module can take a following list of properties:
```

Comment:
"prop" is a specific term in React, see https://reactjs.org/docs/components-and-props.html